docs/ot-2/mkdocs.yml

```
@@ -9,6 +9,10 @@ nav:
   - Labware: labware.md
   - Software: software.md
   - Maintenance: maintenance.md
+  - Appendices:
+      - Additional Documentation: additional-docs.md
+      - Open Source Software: open-source.md
```

```suggestion
      - Open-Source Software: open-source.md
```
Matches Flex nav and the hyphenation within the page itself